homeassistant/components/binary_sensor/knx.py

```diff
@@ -49,6 +50,8 @@
     vol.Optional(CONF_DEVICE_CLASS): cv.string,
     vol.Optional(CONF_SIGNIFICANT_BIT, default=CONF_DEFAULT_SIGNIFICANT_BIT):
         cv.positive_int,
+    vol.Optional(CONF_RESET_AFTER, default=None): vol.Any(
```

Comment:
I'd prefer if you remove the default. You are using `dict.get` below so the default here has no effect.

Reply:
changed it to: `vol.Optional(CONF_RESET_AFTER): cv.positive_int,`